services/movie_session.py

```diff
@@ -42,3 +42,10 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
```

Comment:
The return type hint for `get_taken_seats` should be `list[dict]` to accurately reflect the structure of the returned data.